### ISSUE TITLE
You must specify a valid value for runtime at nodejs8.10

### ISSUE THREAD

**Issue by reidca** [awslabs/cfn-python-lint#191]
cfn-lint --version 
cfn-lint 0.3.1

vscode-cfn-lint: 0.1.2

Getting the error message in the title for this Lambda function:

```
    ECSCleanUpTasksFunction:
      Type: AWS::Lambda::Function
      DependsOn:
        - ECSLambdaRole
        - CustomResourceGetAttHelperS3InfrastructureAutomationBucket
      Properties:
        Code:
          S3Bucket: !GetAtt CustomResourceGetAttHelperS3InfrastructureAutomationBucket.BucketName
          S3Key:
            !Join
            - ''
            - - !Ref S3BucketPathToLambda
              - !Ref ECSCleanUpTasksFunctionName
              - ".zip"
        Environment:
          Variables:
            HostedZoneId:  !FindInMap 
              - ZoneIds 
              - !Ref 'AWS::AccountId'
              - ZoneId        
        FunctionName:
          Ref: ECSCleanUpTasksFunctionName
        Handler:
          !Join
          - ''
          - - !Ref ECSCleanUpTasksFunctionName
            - ".handler"
        MemorySize: 128
        Role: !GetAtt 'ECSLambdaRole.Arn'
        Runtime: nodejs8.10
        Timeout: 10
```

According to the docs https://docs.aws.amazon.com/lambda/latest/dg/API_CreateFunction.html#SSS-CreateFunction-request-Runtime valid values include nodejs8.10

**Comment by fatbasstard** [comment]
Hi! 

This is caused by a deployment issie in 0.3.1. An old testcase was included that should not be in there.

This is resolved in 0.3.2, so if you update cfn-lint this issue should be resolved

**Comment by reidca** [comment]
yep that's fixed, thanks.
